src/extension.ts

```diff
@@ -24,7 +24,8 @@ const criticalWarningTypes = [
     'preprocessorErrorDirective',
     'syntaxError',
     'unhandledChar',
-    'unknownMacro'
+    'unknownMacro',
+    'checkersReport'
```

Comment:
it's just weird to put checkersReport in "criticalWarningTypes" because it's not critical

Reply:
I am skeptic that users wants to see the checkersReport typically. maybe it could be optional.